code/controlconfig/controlsconfig.cpp

```diff
@@ -393,21 +393,10 @@ int control_config_detect_axis()
 
 void control_config_conflict_check()
 {
-	int i, j, shift = -1, alt = -1;
-
+	int i, j;
+	
 	for (i=0; i<CCFG_MAX; i++) {
 		Conflicts[i].key = Conflicts[i].joy = -1;
-		switch (Control_config[i].key_id) {
-			case KEY_LSHIFT:
-			case KEY_RSHIFT:
-				shift = i;
```

Comment:
Not quite. I refactored by removing `control_config_valid_action()`, which was a function that always returned 1.
So this section of code on topic now wasn't really part of that patch, because it was always going to get hit.

Whether it does anything useful at all (being the issue now) is a separate one.

In short, I don't mind seeing this go, but it's not because I have any special knowledge of this function from 2010.